packages/themes/src/hook/index.tsx

```diff
@@ -16,6 +17,16 @@ export const useGetColorScheme = (themeKey: string, variantKey: string) => {
   return variant;
 };
 
+export const useTheme = () => {
```

Comment:
I think you forgot to close the scope for this function and this should be an error, why can it be committed by a husky?

Reply:
hahaha i think that bc i resolve conflict from git editor not from IDE, let me push one sec.